dockstore-webservice/src/main/java/io/dockstore/webservice/CharsetResponseFilter.java

```diff
@@ -39,8 +39,12 @@ public class CharsetResponseFilter implements ContainerResponseFilter {
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) {
         MediaType contentType = responseContext.getMediaType();
         if (contentType != null) {
-            if (!contentType.toString().toLowerCase().contains("charset=utf-8")) {
-                responseContext.getHeaders().putSingle("Content-Type", contentType.toString() + ";charset=UTF-8");
+            boolean isText = "text".equals(contentType.getType());
```

Comment:
To be paranoid, should we have `equalsIgnoreCase` here and on the next line?

Reply:
Done.  I've never seen a non-standardly-capitalized mime type but it could happen.
Next line compares "types" and should do it correctly.